src/webhooks/test/registry.test.ts

```diff
@@ -656,6 +641,54 @@ describe('ShopifyWebhooks.Registry.isWebhookPath', () => {
   });
 });
 
+describe('ShopifyWebhooks.Registry.addHandler', () => {
+  beforeEach(async () => {
+    ShopifyWebhooks.Registry.webhookRegistry = {};
+  });
+
+  it('adds two handlers to the webhook registry', async () => {
+    await ShopifyWebhooks.Registry.addHandler({path: '/webhooks', topic: 'PRODUCTS_CREATE', webhookHandler: genericWebhookHandler});
```

Comment:
Nit: is the linter OK with such long lines? It probably shouldn't be 😅  - we might want to fix that as a separate PR.